<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/Picture-Viewer-icon.ico</normaloff>:/images/images/Picture-Viewer-icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Папка</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Недавние папки</string>
     </property>
     <addaction name="action_empty"/>
     <addaction name="separator"/>
     <addaction name="action_clear_recent_list"/>
    </widget>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Действие</string>
    </property>
    <addaction name="action_mark"/>
    <addaction name="separator"/>
    <addaction name="action_move"/>
    <addaction name="action_copy"/>
    <addaction name="action_remove"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_zoom_in"/>
    <addaction name="action_zoom_out"/>
    <addaction name="separator"/>
    <addaction name="action_fit_to_screen"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_5"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_move"/>
   <addaction name="action_copy"/>
   <addaction name="action_remove"/>
   <addaction name="separator"/>
   <addaction name="action_prev"/>
   <addaction name="action_next"/>
   <addaction name="separator"/>
   <addaction name="action_zoom_in"/>
   <addaction name="action_zoom_out"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Open_24x24.png</normaloff>:/images/images/Open_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="action_empty">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>(пусто)</string>
   </property>
  </action>
  <action name="action_clear_recent_list">
   <property name="text">
    <string>Отчистить</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_mark">
   <property name="text">
    <string>Отметить</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="action_move">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Cut_24x24.png</normaloff>:/images/images/Cut_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Переместить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Copy_24x24.png</normaloff>:/images/images/Copy_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Remove_24x24.png</normaloff>:/images/images/Remove_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="action_next">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Next_24x24.png</normaloff>:/images/images/Next_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Следующий</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="action_prev">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Previous_24x24.png</normaloff>:/images/images/Previous_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Предыдущий</string>
   </property>
   <property name="toolTip">
    <string>Предыдущий</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="action_zoom_in">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Zoom In_24x24.png</normaloff>:/images/images/Zoom In_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_zoom_out">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Zoom Out_24x24.png</normaloff>:/images/images/Zoom Out_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_fit_to_screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Растянуть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
